pyproject.toml

```diff
@@ -6,6 +6,34 @@
         "cython>=0.29.21; python_implementation == 'CPython'", # Skip cython when using pypy
     ]
 
+[tool.mypy]
+    exclude = "falcon/bench/|falcon/cmd/"
+
+    [[tool.mypy.overrides]]
```

Comment:
I don't think you are supposed to indent inner prop. at least the toml formatter I've used will unintended this

Reply:
Most probably you're right, I'm no expert (nor fan for that matter) of TOML... will check.

Reply:
I'm going to leave this as-is as the [specification states](https://github.com/toml-lang/toml#example):
> `# Indentation (tabs and/or spaces) is allowed but not required`